server/cli/launch.py

```diff
@@ -1,9 +1,10 @@
 import logging
 from os import devnull
-from os.path import splitext, basename
+from os.path import splitext, basename, exists, getsize
 import sys
 import warnings
 import webbrowser
+import psutil
```

Comment:
This is a third party import, so we should move to below click

Reply:
will do